### PR TITLE
Fix crash in UserImageView (missing UserSession)

### PR DESCRIPTION
**Pull request by mikeger**
UserSession has to be injected now, since the UserImageView is ported to WireExtensionComponents, and there the user session is not possible to have.

### CODE REVIEW

File: Wire-iOS/Sources/UserInterface/Conversation/ReactionsList/ReactionCell.swift [hunk 1]
```diff
@@ -55,6 +55,8 @@ import Classy
     public override init(frame: CGRect) {
         super.init(frame: frame)
         
+        self.userImageView.userSession = ZMUserSession.shared()
```
Comment (daehn) [inline Wire-iOS/Sources/UserInterface/Conversation/ReactionsList/ReactionCell.swift]: If the `UserImageView` needs the user session in order to request downloading pictures it might make sense to add is as a parameter to the `init`. Could be optional as is won't be there in an extension, but I fear people in the future not knowing they have to set a user session in order for the image downloading to work.

Comment (mikeger) [inline Wire-iOS/Sources/UserInterface/Conversation/ReactionsList/ReactionCell.swift]: @daehn agree, was also thinking how to inject it properly, but we have several `init`s and also sometimes we have magic involved, so I would propose to make an another PR that would remove magic and add the parameter.
